src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -304,6 +304,23 @@ protected static void assertEqualsDoubleArray(final double[] actual, final doubl
             Assert.assertEquals(actual[i],expected[i],tolerance,"array position " + i);
     }
 
+    /**
+     * Checks whether two long arrays contain the same values or not.
+     * @param actual actual produced array.
+     * @param expected expected array.
+     * @param tolerance maximum difference between double value to be consider equivalent.
+     */
+    protected static void assertEqualsLongArray(final long[] actual, final long[] expected) {
```

Comment:
It's annoying that this isn't in testng.  Junit has it, so we could use that, but it's a bit weird to use both testng Assert and junitAssert